proposals/0304-structured-concurrency.md

```diff
@@ -763,30 +760,34 @@ extension Task {
 }
 ```
 
-The functions work the same as their instance counter parts, except that if invoked from a context that has no Task available, e.g. if invoked from outside of Swift's concurrency model (e.g. directly from a pthread) a default value is returned.
+These work the same as their instance counter parts, except that if invoked from a context that has no Task available, e.g. if invoked from outside of Swift's concurrency model (e.g. directly from a pthread) a default value is returned.
 
-The isCancelled function is implemented as:
+The `isCancelled` property is implemented as:
 
 ```swift
 extension Task {
   static var isCancelled: Bool { 
-    Task.current?.isCancelled ?? false
```

Comment:
Another case where the proposal is more up to date than the impl, please keep it.

We don't need to drop to unsafe to implement this.